ui/admin/app/routes/_auth.tsx

```diff
@@ -2,18 +2,26 @@ import { Outlet, isRouteErrorResponse, useRouteError } from "@remix-run/react";
 import { preload } from "swr";
 
 import { ForbiddenError, UnauthorizedError } from "~/lib/service/api/apiErrors";
+import { ModelProviderApiService } from "~/lib/service/api/modelProviderApiService";
 import { UserService } from "~/lib/service/api/userService";
 
 import { useAuth } from "~/components/auth/AuthContext";
+import { FirstModelProviderBanner } from "~/components/composed/FirstModelProviderBanner";
 import { Error, RouteError, Unauthorized } from "~/components/errors";
 import { HeaderNav } from "~/components/header/HeaderNav";
+import { ModelProviderProvider } from "~/components/model-providers/ModelProviderContext";
 import { Sidebar } from "~/components/sidebar";
 import { SignIn } from "~/components/signin/SignIn";
 
 export async function clientLoader() {
-    const me = await preload(UserService.getMe.key(), () =>
-        UserService.getMe()
-    );
+    const promises = await Promise.all([
+        preload(UserService.getMe.key(), () => UserService.getMe()),
+        preload(
+            ModelProviderApiService.getModelProviders.key(),
+            ModelProviderApiService.getModelProviders
+        ),
+    ]);
+    const me = promises[0];
```

Comment:
Wondering if I can drop this `return { me }`? Thought I'd see a 'useLoaderData' reference but maybe I missed it!